sessionctx/variable/sysvar.go

```diff
@@ -825,6 +825,13 @@ var defaultSysVars = []*SysVar{
 	}, GetSession: func(s *SessionVars) (string, error) {
 		return s.TxnScope.GetVarValue(), nil
 	}},
+	{Scope: ScopeSession, Name: TiDBTxnReadTS, Value: "", SetSession: func(s *SessionVars, val string) error {
```

Comment:
Is this sysvar exposed to users?

Reply:
I think not

Reply:
The problem is, users may also use this variable to set read ts.